StabilityPackage.wl

```diff
@@ -602,13 +604,20 @@ Return[out];
 ];
 
 
+(*Get a 10 bin theta grid refined nref times*)

```

Comment:
I like this code reuse! I think this function can be modified to calculate rather than import the walls/middles relatively easily.

Reply:
Yeah I caught myself on this one! I think you're right; I'm just being lazy for the time being. I got caught up working on that for about 45 minutes when my timer went off to remind me I should move on to a more important task.